src/main/java/de/qabel/desktop/ui/accounting/GsonEntity.java

```diff
@@ -0,0 +1,86 @@
+package de.qabel.desktop.ui.accounting;
+
+import com.google.gson.JsonArray;
+import com.google.gson.annotations.Expose;
+
+/**
+ * Created by jmt on 18.01.16.
+ */
+public abstract class GsonEntity {
```

Comment:
Entity doesn't sound too coarse here. It's an Identity / Contact / Personalinformation / don't know...